src/main/webapp/encounters/encounter.jsp

```diff
@@ -674,8 +674,8 @@ var encounterNumber = '<%=num%>';
 
 				//let's see if this user has ownership and can make edits
       			boolean isOwner = ServletUtilities.isUserAuthorizedForEncounter(enc, request,myShepherd);
-            boolean isPublic = ServletUtilities.isEncounterOwnedByPublic(enc) && request.getUserPrincipal() != null; //should at least be logged in
```

Comment:
reading through this file, i think we actually need 2 value: one for something is _publicly displayable_ and one when something is _public and can be edited_. perhaps like this:
```
boolean isPublicViewable = ServletUtilities.isEncounterOwnedByPublic(enc);
boolean isPublicEditable = isPublic && (request.getUserPrincipal() != null);
```

then, below, you will use one or the other of these depending on whether the code is just for **showing values** (`isPublicViewable`) or for **allowing editing** of public data (`isPublicEditable`)